integration-cli/docker_cli_import_test.go

```diff
@@ -29,8 +32,52 @@ func (s *DockerSuite) TestImportDisplay(c *check.C) {
 	image := strings.TrimSpace(out)
 
 	runCmd = exec.Command(dockerBinary, "run", "--rm", image, "true")
-	out, _, err = runCommandWithOutput(runCmd)
+	out, ec, err = runCommandWithOutput(runCmd)
+	if ec != 0 || err != nil {
+		c.Fatal("failed to create a container", out, err)
+	}
+
+	if out != "" {
+		c.Fatalf("command output should've been nothing, was %q", out)
+	}
+
+}
+
+func (s *DockerSuite) TestImportFile(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--name", "test-import", "busybox", "true")
+	out, ec, err := runCommandWithOutput(runCmd)
+	if ec != 0 || err != nil {
```

Comment:
I apologize for this - I totally misread the code in runCommandWithOutput() and was corrected on this in another PR, apparently err can't be nil if ec != 0, so what you had originally (just checking err) is good enough.  Really sorry for my mistake.